pkg/converter/convert.go

```diff
@@ -217,6 +217,28 @@ func Convert(o Options, pathOptions *clientcmd.PathOptions) error {
 				exec.Args = append(exec.Args, argIsLegacy)
 			}
 
+		case token.InteractiveLogin:
```

Comment:
add unit test please

Reply:
Working on it and unit test will show up in next commit.

Reply:
Added unit test for interactive cases in convert_test.go in the most recent commit e916641. Please check it out. Thanks!